### DISCUSSION TITLE
Camera and Crouching

### DISCUSSION THREAD

**Discussion by thelightxen** [ProjectBorealis/PBCharacterMovement#49] [Q&A]
Hi, 
When adding a camera component to a player, the camera does not move correctly when crouching.

I fixed this (in my project) using DetachFromParent, teleporting player to the world camera location and again AttachToComponent for the camera.
But what if this can be fixed differently?

**Comment by mastercoms** [comment]
use player camera manager instead of a camera component

**Reply by thelightxen** [reply]
what if I need to use SpringArm or attach something to the camera?

**Reply by mastercoms** [reply]
camera manager should support any set up like that.

if that doesn't fix it, i do know there's specific rare bugs with non uniform eye height that I'll be pushing the fix for at some point
